### PR TITLE
improve boskos logging, and downgrade unnecessary errors to info/debu…

### PR DESCRIPTION
**Pull request by krzyzacy**
…g level

Also made janitor call update while cleaning.

/assign @fejta @cjwagner @stevekuznetsov @sebastienvas @amwat

### CODE REVIEW

**Comment by fejta** [comment]
/lint
Looks like CLeaning is misspelled. Potentially other errors

File: boskos/janitor/janitor.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ import (
 	"os/exec"
 	"strings"
 	"time"
-
+g
```
Comment (stevekuznetsov) [inline boskos/janitor/janitor.go]: :(

Comment (krzyzacy) [reply boskos/janitor/janitor.go]: ehhh

Comment (krzyzacy) [reply boskos/janitor/janitor.go]: :chaosmonkey:

**Comment by krzyzacy** [comment]
fixed tests, PTAL

**Pull request merged**